packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -139,4 +138,183 @@ startTelemetry(telemetryConfig);
 // Done! Your container telemetry is now being created and sent to your Telemetry Consumer
 ```
 
-You can now run the app and see the telemetry being printed on your console.
\ No newline at end of file
+You can now run the app and see the telemetry being printed on your console.
+
+## <a name="telemetry_visualization">Interpreting telemetry data</a>
```

Comment:
Oh, interesting, is this a standard thing with markdown? I would have used `[link text](#interpreting-telemetry-data)` above but fixing an id seems way better.

Reply:
yup! I learnt it today also. Was not aware previously; this comes in handy.